LION/optimizers/LIONsolver.py

```diff
@@ -45,6 +46,19 @@ class SolverParams(LIONParameter):
     def __init__(self):
         super().__init__()
 
+def normalize_input(func):
+    def wrapper(self, *inputs):
+        if self.do_normalize:
+            print(inputs)
```

Comment:
Lets not print always. This is why self.verbose exist in LIONSolver, for example.

Reply:
don't think it ever needs printing, that was for debugging 🤦 
fixed.